### PR TITLE
Enable PHP 7.3 tests and functional for PHP 7.2

### PR DESCRIPTION
**Pull request by snapshotpl**

### CODE REVIEW

**Comment by snapshotpl** [comment]
Finally works!

**Comment by snapshotpl** [comment]
@makasim https://github.com/php-enqueue/enqueue-dev/pull/845 you have duplicate my work ;(

**Comment by makasim** [comment]
Sorry about that. I did not realize that.

File: .travis.yml [hunk 1]
```diff
@@ -7,33 +7,52 @@ matrix:
...
-      env: SYMFONY_VERSION=4.0.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
+    - php: 7.3
+      sudo: false
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
     - php: 7.1
       services: docker
       sudo: required
-      env: SYMFONY_VERSION=3.4.* FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+      env: SYMFONY_VERSION=^3.4 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
```
Comment (makasim) [inline .travis.yml]: The version should be `x.y.*` format to ensure that only that version is allowed. Composer could install anew newer version for `^3.4` constraint.

File: .travis.yml [hunk 2]
```diff
@@ -7,33 +7,52 @@ matrix:
...
+      env: SYMFONY_VERSION=^4.0 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+    - php: 7.2
+      services: docker
+      sudo: required
+      env: SYMFONY_VERSION=^3.4 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+    - php: 7.2
+      sudo: required
+      services: docker
+      env: SYMFONY_VERSION=^4.0 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+    - php: 7.3
```
Comment (makasim) [inline .travis.yml]: Php version does not matter for functional tests because they are run in the container which has php 7.1 inside it.

**Comment by makasim** [comment]
done in https://github.com/php-enqueue/enqueue-dev/pull/845

**Pull request closed without merging**